backend/gsr_booking/models.py

```diff
@@ -96,8 +101,13 @@ class GSR(models.Model):
     name = models.CharField(max_length=255)
     image_url = models.URLField()
 
+    in_use = models.BooleanField(default=True)
+
+    objects = GSRManager()
+    all_objects = models.Manager()  # for admin page
```

Comment:
that is great, thanks vincent! @tuneerroy @ashleyzhang01 @JeffersonDing @dr-Jess just for reference look at this man!